Add reference to `bewcloud-nixos` to README.md (#126)
bewcloud/bewcloud: README.md

>
> `1993:1993` above comes from deno's [docker image](https://github.com/denoland/deno_docker/blob/2abfe921484bdc79d11c7187a9d7b59537457c31/ubuntu.dockerfile#L20-L22) where `1993` is the default user id in it. It might change in the future since I don't control it.

> See the [Community Links](#community-links) section for alternative way of running bewCloud yourself; please be aware that bewCloud developers only test it using Docker though.

Reviewer: ```suggestion
See the [Community Links](#community-links) section for alternative ways of running bewCloud yourself; please be aware these are not officially endorsed.
```

## How does it look?

[Check the website](https://bewcloud.com) for screenshots or [the YouTube channel](https://www.youtube.com/@bewCloud) for 1-minute demos.

## Community Links

>  * [`bewcloud-nixos`](https://gitlab.com/ntninja/bewcloud-nixos/) by @ntninja exposes bewCloud as a NixOS integration as an alternative to using Docker.

Author: Given that there is no guidance on how to run the app locally, I kinda assumed that wasn’t supported. :woman_shrugging:

>  * [`bewcloud-nixos`](https://gitlab.com/ntninja/bewcloud-nixos/) by @ntninja exposes bewCloud as a NixOS integration as an alternative to using Docker.

Author: I only see that mentioned in the “Development” section, that is not end-user guidance.

I would have expected there to be a separate user/administrator-oriented section under “Self-host it!” below the Docker section, telling you how to, well, use it without Docker.

My integration ended up being based entire on the contents of Docker files (both build and compose), since I read the part containing `make start` as being “for development deployments only” and hence skipped over it without reading any of it.

>  * [`bewcloud-nixos`](https://gitlab.com/ntninja/bewcloud-nixos/) by @ntninja exposes bewCloud as a NixOS integration as an alternative to using Docker.

Author: Also, I ended up writing a custom instance initialization script as part of my work.

I’m pretty sure with some minor changes that’d be useful for all new users of bewCloud: https://gitlab.com/ntninja/bewcloud-nixos/-/blob/main/modules/bewcloud/write-secrets-file?ref_type=heads

Are you perhaps interested in including this?

>  * [`bewcloud-nixos`](https://gitlab.com/ntninja/bewcloud-nixos/) by @ntninja exposes bewCloud as a NixOS integration as an alternative to using Docker.

Author: Created #130 to actually document that development setups are supported then (with advanced user warnings)
